scss/bootstrap/overrides/crm/_error-label.scss

```diff
@@ -0,0 +1,5 @@
+// Displays labels marked with errors as they normally would:
+.crm-error-label {
```

Comment:
Why did the label had this weird look in the first place? Was it a regression issue introduced some time ago?

Reply:
That definition comes from [this file](https://github.com/compucorp/org.civicrm.shoreditch/blob/33b3b4da32f06af893da2df1e6b359faad957668/scss/civicrm/common/_forms.scss#L727)

This was originally introduced in [this PR](https://github.com/civicrm/civihr/pull/1656). It seems the style was meant for `.crm-error` elements only (as the screenshots show), but the crm applies that class to labels with errors too.

There seems to be a definition to make labels look as they normally should, but it's not applied (I suspect CSS priority issue):
```scss
label.crm-error {
    background: none;
    color: $gray-darker;

    span {
      background: none;
      color: $gray-darker;
    }
}
```

Reply:
Should I go ahead and apply the fix on this file?

Reply:
So shouldn't you improve the solution that was introduced on https://github.com/civicrm/civihr/pull/1656 instead of adding some new style that counteracts it?